tests/model/test_variant.py

```diff
@@ -27,4 +27,25 @@ def test_get_hgvs_cdna_by_tx(
     ):
         hgvs = some_variant.get_hgvs_cdna_by_tx_id(transcript_id=tx_id)
 
-        assert hgvs == expected
\ No newline at end of file
+        assert hgvs == expected
+        
+    @pytest.mark.parametrize(
+        "search, expected",
+        [
+            (56002673, "12_56002674_56002674_T_C"),
+            (78386857, "16_78386858_78425054_--38197bp--_A"),
+            ('SO:1000029', 'SO:1000037_HGNC:21316_ANKRD11')
+        ] # Couldn't find an example that produces a variant key similar to this: 22_10001_20000_INV
```

Comment:
We probably do not have such variant in *SUOX*. However, it is not too hard to create a `VariantCoordinates` instance that is OK for testing.

Please see `TestVariantCoordinates::test_variant_key` test function and add tests for the other cases.